cli/command/stack/deploy.go

```diff
@@ -252,9 +252,13 @@ func convertServiceNetworks(
 
 	nets := []swarm.NetworkAttachmentConfig{}
 	for networkName, network := range networks {
+		var aliases []string
+		if network != nil {
+			aliases = append(network.Aliases, name)
+		}
```

Comment:
We always want to add the `name` alias, so I think this needs to be `if network == nil { // initialize network }`